packages/bootstrap/scss/_variables.scss

```diff
@@ -1029,6 +1029,28 @@ $dialog-stretched-button-padding-y: $button-padding-y !default;
 $dialog-stretched-button-border-width: $button-border-width !default;
 
 
+// Scrollview
+$scrollview-pagebutton-size: 8px !default;
+$scrollview-pagebutton-bg: rgba($carousel-indicator-active-bg, .5) !default;
+$scrollview-pagebutton-border: null !default;
+$scrollview-pagebutton-primary-bg: null !default;
+$scrollview-pagebutton-primary-border: null !default;
+$scrollview-pagebutton-shadow: 0 0 2px rgba(0, 0, 0, .08) !default;
+
+$scrollview-pager-offset: 20px !default;
+$scrollview-pager-item-spacing: 20px !default;
+$scrollview-pager-item-border-width: 0 !default;
+$scrollview-pager-height: add-three($scrollview-pagebutton-size, 2 * $scrollview-pager-item-border-width, $scrollview-pager-item-spacing / 2 ) !default;
+
+$scrollview-navigation-size: 4.5em !default;
```

Comment:
This is not very descriptive name, why did you change the previous ` $scrollview-arrow-icon-size`?